test/package.json

```diff
@@ -4,15 +4,20 @@
   "description": "test",
   "main": "index.js",
   "license": "MIT",
+  "private": true,
   "dependencies": {
     "react": "*",
     "react-native": "*"
   },
   "devDependencies": {
+    "@babel/helper-hoist-variables": "^7.24.7",
     "@callstack/eslint-config": "link:..",
-    "eslint": "*",
-    "jest": "*",
+    "jest": "^29",
```

Comment:
what's the reason for not being compatible with lower versions?

Reply:
Right, I must've changed it while testing to see if eslint v9 picks the version automatically when specified explicitly, but it didn't (that's why I placed an explicit settings object for jest in `eslint.config.mjs`). Reverted in 0bf92a67b4eb6b228259571082794761c3a161d0